draft-ietf-quic-transport.md

```diff
@@ -3354,7 +3354,9 @@ ih:
   packets.
 
 Section 4 of {{QUIC-TLS}} provides more detail about these restrictions.  Note
-that all frames can appear in 1-RTT packets.
+that all frames can appear in 1-RTT packets.  An endpoint MUST treat receipt of
+a frame in a packet type that is not permitted as a connection error of type
+PROTOCOL_VIOLATION.
```

Comment:
Would FRAME_ENCODING_ERROR be ok too?

Reply:
Interesting question.  It's not something that can be determined without context, so I would lean toward "no".

Reply:
Depends on what your definition of "context" is. If it is "the bytes that this frame is composed of", then no. If you include the encryption level you received those bytes with, then "yes". You might want to do that, because the encryption level is not connection-level state.